projects/packages/blaze/changelog/fix-dsp-stats endpoing

```diff
@@ -0,0 +1,5 @@
+Significance: patch
```

Comment:
filename is "endpoing". I guess it was "endpoint", it also has some space, not sure about the nomenclature, maybe not important

Reply:
i also spotted and fixed it, the push never completed.. but it is fixed now :)